### DISCUSSION TITLE
Drop DESCRIPTION on sync

### DISCUSSION THREAD

**Discussion by devvv4ever** [bitfireAT/icsx5#262] [Ideas]
This was an idea reported by a user via Ticket #4080. He is currently using an online calendar with ICSx⁵. Some online calendars tend to pre-fill the description field with unneccesary "spam". So it would be helpful to ignore the DESCRIPTION upon syncing via an option. It would result in a much cleaner calendar view.

What do you think? @sunkup @ArnyminerZ

**Comment by ArnyminerZ** [comment]
We can add a checkbox to disable `DESCRIPTION` sync in the subscription edit page.

Maybe it's a good moment to add a section called "advanced" or something like that in which we can add these kind of features. There are requests like this every once in a while.

**Reply by sunkup** [reply]
An "advanced" section sounds reasonable to me too :+1:

**Reply by ArnyminerZ** [reply]
I can work on this. Maybe make the section collapsible or something like that? So that the UI is not super cluttered
